### ISSUE TITLE
Code Editor pop up width overflow issue

### ISSUE THREAD

**Issue by tuutoo** [ToolJet/ToolJet#4853]
### Version Information
v1.29.1
### Environment
Docker

### What is the expected behaviour?
This pop-up width should not have width over the screen.

### What is the current behaviour?
This pop-up width have width over the screen and cannot adjust it back.

### How to reproduce the issue?

1. Do a preview or search of the query
2. Then Press "Pop out code editor into a new window" in table property
3. Maybe because of the data have too many data or columns? 

### Screenshots or Screencast
![image](https://user-images.githubusercontent.com/9068314/204283360-103b3ee4-fc2a-4ac6-8545-92f71fb93138.png)

![image](https://user-images.githubusercontent.com/9068314/204282329-dd3d4e07-8e10-43fd-917c-b6abdc91a089.png)

![image](https://user-images.githubusercontent.com/9068314/204282157-8ac9b68f-d326-4dfa-8222-aafb0837704c.png)

### Please provide any traces or logs that could help here.
My Data have over 40 columns at least.

### Can you identify the location in the driver source code where the problem exists?
No.

### If the bug is confirmed, would you be willing to submit a PR?
No. Sorry, I'm not familiar with React...

**Comment by Gauravpawar3102** [comment]
looks like everything is working fine i have checked it

**Comment by tuutoo** [comment]
No, it's not. I have updated to latest version which is 1.30.0, and this issue is still there.
If this issue fixed, there should be a program change linked to this issue...
![image](https://user-images.githubusercontent.com/9068314/206997549-411d5d79-e6e5-49f1-ba7c-0fe051b427cf.png)

**Comment by withshubh** [comment]
Thanks for creating the issue. I have replicated the issue. We will soon be fixing it.

**Comment by tuutoo** [comment]
> Thanks for creating the issue. I have replicated the issue. We will soon be fixing it.

Great! Thanks a lot.

**Comment by DevangAgarwal93** [comment]
Issue no longer exists
